docs/installation.md

```diff
@@ -92,8 +92,17 @@ dependencies {
 To use the C library please get it from your distributions package manager (where available) or build the library from source.
 
 </TabItem>
-</CodeTabs>
+<TabItem value="rs">
+
+To use etebase add the etebase crate to the projects `Cargo.toml`
+
+```toml
+[dependencies]
+etebase="..."
```

Comment:
Could you maybe add a comment that people should set the version and show an example? I know that Rust people will most likely already know that, but I think it will be more "noob-proof". What do you think?

Reply:
Sure, I didn't want to add a version as it's likely to get out of date soon.
Adding it as comment should be fine and we could also add the crates.io badge [![crates.io](https://meritbadge.herokuapp.com/etebase)](https://crates.io/crates/etebase) to the description before the code block

Reply:
That's a really good idea! After we merge this I'll do it for all the rest of the libraries!

Maybe say something like use the version from above (needs better wording), for example: `"^0.4.1"` so people know to use ^? I'd rather they don't pin too harshly so that they get bugfixes.

Reply:
Done